template.go

```diff
@@ -13,73 +39,56 @@ VERSION:
    {{.Version}}{{end}}{{end}}{{if .Description}}
 
 DESCRIPTION:
-   {{wrap .Description 3}}{{end}}{{if len .Authors}}
+   {{template "descriptionTemplate" .}}{{end}}{{if len .Authors}}
```

Comment:
It is possible to strip whitespace between tags https://pkg.go.dev/text/template#hdr-Text_and_spaces
```suggestion
   {{template "descriptionTemplate" .}}{{end}}
{{- if len .Authors}}
```